wordpress-mobile/WordPress-iOS
# Removes the Menus configuration functionality

Fixes #16413

Related PRs:
* `WordPressKit-iOS`: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/398
* `WordPress-iOS-Shared`: https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/293

### Description
Removes the Menus configuration functionality. This includes removing:
* the menu item from *My Sites* screen
* everything under `WordPress/Classes/ViewRelated/Menus/`
* `MenusService.h/m` and `MenusServiceTests.m`

To test:
1. Open the app and select the *My Site* screen
2. Scroll down to the *PERSONALIZE* section
3. **Verify** that only the *Themes* option is available

|Before|After|
|---|---|
|![IMG_B28718BCB92A-1](https://user-images.githubusercontent.com/304044/117803667-910b3480-b25f-11eb-8b40-41a919c5aed9.jpeg)|![IMG_BC6DFED6449B-1](https://user-images.githubusercontent.com/304044/117803655-8f417100-b25f-11eb-9628-81cc95c6416b.jpeg)|

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
4. N/A

PR submission checklist:
